app/MMB/src/navigation/TravelDocumentModalScreen.js

```diff
@@ -53,19 +54,30 @@ export default class TravelDocumentModalScreen extends React.Component<
     }
     return {
       headerLeft: (
-        <HeaderButton onPress={goBack}>
-          <Translation id="mmb.missing_informaiton.travel_document_modal_screen.close" />
-        </HeaderButton>
+        <HeaderButton.Left onPress={goBack}>
+          {Platform.OS === 'android' ? (
+            <Icon name="close" size={24} color={Color.brand} />
+          ) : (
```

Comment:
Could this somehow be done in the component instead, so we don't have to do this with all header buttons?